callisto/delivery/report_delivery.py

```diff
@@ -244,11 +247,24 @@ def render_question(self, question):
                     for q in form:
                         self.render_question(q)
                     self.pdf_elements.append(Indenter(left=-12))
-            else:
-                self.add_question(prompt)
-                self.pdf_elements.append(Indenter(left=12))
-                self.add_question('<i>None added</i>')
-                self.pdf_elements.append(Indenter(left=-12))
+
+        def _render_default():
+            prompt = question.get('prompt').capitalize()
+            self.add_question(prompt)
+            self.pdf_elements.append(Indenter(left=12))
+            self.add_question('<i>None added</i>')
+            self.pdf_elements.append(Indenter(left=-12))
+
+        render_functions = {
```

Comment:
Just to make sure I didn't miss something further: I did miss Date here, but I do have RadioButton in the render_function dict. I put the keys in alpha order, so it's not next to Checkbox, but it still will call _render_multiple_choice. Right?